pkgs/racket-doc/scribblings/reference/pairs.scrbl

```diff
@@ -875,79 +875,119 @@ The same as @racket[(car lst)], but only for lists (that are not empty).
 The same as @racket[(cdr lst)], but only for lists (that are not empty).
 
 @mz-examples[#:eval list-eval
-  (rest '(1 2 3 4 5 6 7 8 9 10))]}
+  (rest '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(second [lst list?]) any/c]{
 
 Returns the second element of the list.
 
 @mz-examples[#:eval list-eval
-  (second '(1 2 3 4 5 6 7 8 9 10))]}
+  (second '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(third [lst list?]) any/c]{
 
 Returns the third element of the list.
 
 @mz-examples[#:eval list-eval
-  (third '(1 2 3 4 5 6 7 8 9 10))]}
+  (third '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(fourth [lst list?]) any/c]{
 
 Returns the fourth element of the list.
 
 @mz-examples[#:eval list-eval
-  (fourth '(1 2 3 4 5 6 7 8 9 10))]}
+  (fourth '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(fifth [lst list?]) any/c]{
 
 Returns the fifth element of the list.
 
 @mz-examples[#:eval list-eval
-  (fifth '(1 2 3 4 5 6 7 8 9 10))]}
+  (fifth '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(sixth [lst list?]) any/c]{
 
 Returns the sixth element of the list.
 
 @mz-examples[#:eval list-eval
-  (sixth '(1 2 3 4 5 6 7 8 9 10))]}
+  (sixth '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(seventh [lst list?]) any/c]{
 
 Returns the seventh element of the list.
 
 @mz-examples[#:eval list-eval
-  (seventh '(1 2 3 4 5 6 7 8 9 10))]}
+  (seventh '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(eighth [lst list?]) any/c]{
 
 Returns the eighth element of the list.
 
 @mz-examples[#:eval list-eval
-  (eighth '(1 2 3 4 5 6 7 8 9 10))]}
+  (eighth '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(ninth [lst list?]) any/c]{
 
 Returns the ninth element of the list.
 
 @mz-examples[#:eval list-eval
-  (ninth '(1 2 3 4 5 6 7 8 9 10))]}
+  (ninth '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
 
 
 @defproc[(tenth [lst list?]) any/c]{
 
 Returns the tenth element of the list.
 
 @mz-examples[#:eval list-eval
-  (tenth '(1 2 3 4 5 6 7 8 9 10))]}
+  (tenth '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
+
+
+@defproc[(eleventh [lst list?]) any/c]{
+
+Returns the eleventh element of the list.
+
+@mz-examples[#:eval list-eval
+  (eleventh '(1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))]}
```

Comment:
The documentation for each new function needs a `history` note to say when it was added: `@history[#:added "8.15.0.3"]` before the closing `}`.